src/qt/walletcontroller.cpp

```diff
@@ -57,13 +58,33 @@ std::vector<std::string> WalletController::getWalletsAvailableToOpen() const
     return wallets;
 }
 
+bool WalletController::checkWalletExists(const std::string name) const
```

Comment:
string&

Reply:
Not sure if this check (as it is pays) off. It doesn't check if name as a path exists - since name can be a path. Maybe add `m_node.canCreateWallet(name)`?

Reply:
I've removed the check entirely since the same thing is done in the `CreateWallet()` function already.